ccm_web/client/src/components/InlineTextEdit.tsx

```diff
@@ -69,6 +79,11 @@ function InlineTextEdit (props: InlineTextEditProps): JSX.Element {
     }
   }
 
+  const error = validationResult !== undefined && !validationResult.isValid
+  const helpMessage = validationResult !== undefined && validationResult.messages.length > 0
+    ? validationResult.messages[0]
+    : undefined
```

Comment:
So in a situation when the `helpMessage` `IF` statement is false, `undefined` is picked up. Why undefined? and why can you have some custom message as you did [here](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/267/files#diff-eb168987f5c7eaf6d186216d62d3555de5cd7706fd48e8cc06e8d01c8df2d3caR64)

Reply:
We could add a custom message there. I guess I figured that case is not likely to ever happen, and if it did, the field would still have the red border styling indicating it's an error. I don't mind adding a fallback though if you want, something like "The value for course name is invalid."

Reply:
Yeah.. lets have some default message as a fallback

Reply:
Okay, I tweaked this a bit here: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/267/commits/66aa65cb757663131dd04b7fec9335de0d04a4d5 I realized that I wanted to keep the message general since theoretically we may want to use `InlineTextEdit`. Also, since the fallback is an invalid message, I added a check to make sure the result is invalid, and just not show anything if it is. As of now, this is a non-issue, since I only store the validation result if it is invalid. I think what I added should be okay for now, this should be all edge edge case handling.